packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -184,6 +184,48 @@ const product = new servicecatalog.CloudFormationProduct(this, 'Product', {
 });
 ```
 
+We can also define a `versioningStrategy` for your `ProductStack` deployment. 
```

Comment:
Lead with the use case instead of the mechanism. WHY would a user want to read this section? Also explain how it works in general, because this has implications on the user's workflow, not just on the API they need to call. Add a section header.

I'd recommend something like:

```
## Immutable products

The default behavior of Service Catalog is to overwrite each product version upon deployment.
If instead you want to never overwrite existing versions, but only add new versions... (etc)

...all product templates will be written to disk, so that they will still be available in the future
as the definition of the `ProductStack` subclass changes over time. **It is very important** that you commit these 
old versions to source control...
```